bindings/pyroot_experimental/PyROOT/python/ROOT/pythonization/_rtensor.py

```diff
@@ -0,0 +1,59 @@
+from libROOTPython import GetEndianess, GetVectorDataPointer, GetSizeOfType, AddRTensorGetSetItem
+from ROOT import pythonization
+from ROOT.pythonization._rvec import _array_interface_dtypes, _array_interface_dtype_map
+
+
+def get_array_interface(self):
```

Comment:
Perhaps we could add some short documentation also to this function and the two below, in the form of comments (not docstring, since they are private), just to explain briefly what they do and what are their parameters.

Reply:
curious: why don't you want docstrings for private functions?

Reply:
Here's what PEP 8 says about it:
https://www.python.org/dev/peps/pep-0008/

*Write docstrings for all public modules, functions, classes, and methods. Docstrings are not necessary for non-public methods, but you should have a comment that describes what the method does. This comment should appear after the "def" line.*

In the end, we can do whatever we want :) Using comments for private methods is one of the options. It is also a way to distinguish between public and private.